### PR TITLE
chore: improved README

### PR DESCRIPTION
**Pull request by sum2000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,6 +112,33 @@ When using a JSON logger, the output should look something like this:
 }
 ```
 
+## Formatted error printing
+The default format in `eris` is returned by the method [`NewDefaultFormat()`](https://godoc.org/github.com/morningvera/eris#NewDefaultFormat).Below you can see the what the default formatted error in `eris` might look like.
```
Comment (morningvera) [inline README.md]: Missing space after link

File: README.md [hunk 2]
```diff
@@ -112,6 +112,33 @@ When using a JSON logger, the output should look something like this:
...
+
+Errors printed without trace using `fmt.Printf("%v\n", err)`
+```golang
```
Comment (morningvera) [inline README.md]: shouldn't be golang

File: README.md [hunk 3]
```diff
@@ -112,6 +112,33 @@ When using a JSON logger, the output should look something like this:
...
+even more context: additional context: root error
+```
+
+Errors printed with trace using `fmt.Printf("%+v\n", err)`
+```golang
```
Comment (morningvera) [inline README.md]: shouldn't be golang

File: README.md [hunk 4]
```diff
@@ -112,6 +112,33 @@ When using a JSON logger, the output should look something like this:
...
+additional context
+        eris_test.setupTestCase: ../eris/eris_test.go: 17
+root error
+        eris_test.setupTestCase: ../eris/eris_test.go: 17
+        eris_test.TestErrorFormatting: ../eris/eris_test.go: 226
+        testing.tRunner: ../go1.11.4/src/testing/testing.go: 827
+        runtime.goexit: ../go1.11.4/src/runtime/asm_amd64.s: 1333
+```
+
+The 'eris' also provides developers a way to define custom format to print the errors. The [`Format`](https://godoc.org/github.com/morningvera/eris#Format) object defines separators for various components of the error/trace and can be it passed to utility methods for printing string and JSON formats.
```
Comment (morningvera) [inline README.md]: `eris`, not "The 'eris'"

Comment (morningvera) [inline README.md]: typo "error/trace and can be it passed" -> "error/trace and can be passed"

File: README.md [hunk 5]
```diff
@@ -112,6 +112,33 @@ When using a JSON logger, the output should look something like this:
...
 
+## Formatted error printing
+The default format in `eris` is returned by the method [`NewDefaultFormat()`](https://godoc.org/github.com/morningvera/eris#NewDefaultFormat).Below you can see the what the default formatted error in `eris` might look like.
+
+Errors printed without trace using `fmt.Printf("%v\n", err)`
+```golang
+even more context: additional context: root error
+```
+
+Errors printed with trace using `fmt.Printf("%+v\n", err)`
```
Comment (morningvera) [inline README.md]: Leave a space between text lines and code blocks

File: README.md [hunk 6]
```diff
@@ -76,6 +76,32 @@ not found
 
 The first layer of the full error output shows a message ("error getting resource 'example-id'") and a single stack frame. The next layer shows the root error ("not found") and the full stack trace.
 
+
+## Formatted error printing
+The default format in `eris` is returned by the method [`NewDefaultFormat()`](https://godoc.org/github.com/morningvera/eris#NewDefaultFormat). Below you can see the what the default formatted error in `eris` might look like.
```
Comment (morningvera) [inline README.md]: typo "you can see the what" -> "you can see what"

File: README.md [hunk 7]
```diff
@@ -76,6 +76,32 @@ not found
...
+
+Errors printed without trace using `fmt.Printf("%v\n", err)`
+```
+even more context: additional context: root error
+```
+
+Errors printed with trace using `fmt.Printf("%+v\n", err)`
+```
+additional context
```
Comment (morningvera) [inline README.md]: this is missing the "even more context" line, probably makes sense to keep them consistent

File: README.md [hunk 8]
```diff
@@ -112,6 +138,7 @@ When using a JSON logger, the output should look something like this:
 }
 ```
 
+
```
Comment (morningvera) [inline README.md]: Random extra line here

File: README.md [hunk 9]
```diff
@@ -34,7 +34,7 @@ if err != nil {
 
 ## Inspecting error types
 
-The `eris` package provides a few ways to inspect and compare error types. [`eris.Is`](https://godoc.org/github.com/rotisserie/eris#Is) returns true if a particular error appears anywhere in the error chain, and `eris.Cause` returns the root cause of the error. Currently, `eris.Is` works simply by comparing error messages with each other. If an error contains a particular message anywhere in its chain (e.g. "not found"), it's defined to be that error type (i.e. `eris.Is` will return `true`).
+`eris` package provides a few ways to inspect and compare error types. [`eris.Is`](https://godoc.org/github.com/rotisserie/eris#Is) returns true if a particular error appears anywhere in the error chain, and `eris.Cause` returns the root cause of the error. Currently, `eris.Is` works simply by comparing error messages with each other. If an error contains a particular message anywhere in its chain (e.g. "not found"), it's defined to be that error type (i.e. `eris.Is` will return `true`).
```
Comment (morningvera) [inline README.md]: The `eris` package made more sense

Comment (sum2000) [reply README.md]: LOL

Comment (sum2000) [reply README.md]: i searched and replaced

File: README.md [hunk 10]
```diff
@@ -76,6 +76,36 @@ not found
 
 The first layer of the full error output shows a message ("error getting resource 'example-id'") and a single stack frame. The next layer shows the root error ("not found") and the full stack trace.
 
+
```
Comment (morningvera) [inline README.md]: Random extra line

File: README.md [hunk 11]
```diff
@@ -76,6 +76,36 @@ not found
...
+## Formatted error printing
```
Comment (morningvera) [inline README.md]: Add line break between header and text

**Pull request merged**